src/DeVinci_frontend/helpers/ai_model_helpers.js

```diff
@@ -255,3 +279,13 @@ export const initiateCollapsibles = () => {
     coll[i].addEventListener('click', addCollapsibleFunctionality);
   };
 };
+
+function parametersToGB(parametersStr) {
+  const number = parseFloat(parametersStr);
+  if (parametersStr.includes('billion')) {
+    return (number * 2).toFixed(1);  // Each billion parameters is roughly 2GB

```

Comment:
this is not really true. I'm not sure we can generalize this computation. We can take another look at the actual sizes and parameter numbers though

Reply:
yes not exact, i tested some and the approximation is good. how else do you suggest we can do? did not found the size of the models anywhere, otherwise we need to download one by one and hardcore the values.

Reply:
it might be ok for unquantized models but most of the models are quantized, so we'll indicate way too high numbers. The best value we have is the vram requirement and it's best to just display that value as the model's size In GB).

Reply:
alright, let's remove that indicator. perhaps later we can find a way to find those values as I find it useful for the user to know how much GB they have to download.

Reply:
let's use the vram requirement as that's a great approximation for the download size and is useful to know as well for memory constraints. Likely many users won't know/care about the difference and putting two values there would be a bit much and potentially confusing. We could e.g. just show the vram requirement value as Model Size or Total Size (and if we want to show additional info on the number, say that it's the size of the files to download and run which the user needs to make sure the device supports). How does this sound?

Reply:
ok i cleaned up the function to display only one value

Reply:
nice, thank you